### PR TITLE
Gutenboarding: update copy and punctuation

### PR DESCRIPTION
**Pull request by razvanpapadopol**
#### Changes proposed in this Pull Request
- Use copy suggested in p58i-8QA-p2#comment-45974
- Use period at the end of subtitles.
- Use period at the end of site title input when there is a non-empty value.

#### Testing instructions

* go to `/new`
* check the copy/punctuation updates mentioned above
* Editor welcome modal changes can be reviewed by following FSE development guide: PCYsg-ly5-p2

### CODE REVIEW

**Comment by benhuberman** [comment]
Hi @razvanpapadopol  -- I wanted to respond to the `Needs copy review` request, but I wasn't sure where to find the copy in question (or how to access /new in a way that would show it). Could you share how best to view it, and/or share screenshots here? Thanks in advance!

File: apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js [hunk 1]
```diff
@@ -65,24 +65,32 @@ function WpcomNux() {
 			<NuxPage
 				heading={ welcomeHeading }
 				description={ __(
-					'Create and edit site pages, and customize the look and feel of each page.'
+					'Edit your homepage and add the pages you need, change the look and feel of everything.'
```
Comment (razvanpapadopol) [inline apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js]: Screenshot 2020-04-14 at 15 56 20

File: apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js [hunk 2]
```diff
@@ -65,24 +65,32 @@ function WpcomNux() {
...
+						? __( 'Customize your content' )
+						: __( 'Create pages and add your content' )
+				}
+				description={
+					isGutenboarding
+						? __( 'Start with a pre-made layout and make it your own.' )
+						: __(
+								'Create and rearrange your site pages. Customize your site navigation menus so your visitors can explore your site.'
+						  )
+				}
```
Comment (razvanpapadopol) [inline apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js:90]: Screenshot 2020-04-14 at 15 54 18

File: apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js [hunk 3]
```diff
@@ -65,24 +65,32 @@ function WpcomNux() {
...
 				imgSrc={ blockImage }
 			/>
 
 			<NuxPage
 				heading={ __( 'Add anything you want' ) }
 				description={ __(
-					'Insert text, photos, forms, Yelp reviews, testimonials, maps, and many more types of blocks. Rearrange blocks on your pages to meet your needs.'
+					'Insert text, photos, forms, Yelp reviews, testimonials, maps, and more. Rearrange the blocks on your page till it’s just right.'
```
Comment (razvanpapadopol) [inline apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js:97]: Screenshot 2020-04-14 at 15 55 00

File: apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js [hunk 4]
```diff
@@ -64,25 +64,37 @@ function WpcomNux() {
...
-				description={ __(
-					'Create and edit site pages, and customize the look and feel of each page.'
-				) }
+				description={
+					isGutenboarding
+						? __(
+								'Edit your homepage and add the pages you need, change the look and feel of everything.'
+						  )
+						: __( 'Create and edit site pages, and customize the look and feel of each page.' )
+				}
```
Comment (razvanpapadopol) [inline apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js:73]: Screenshot 2020-04-14 at 15 56 20

File: apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js [hunk 5]
```diff
@@ -52,7 +52,7 @@ function WpcomNux() {
 
 	const isGutenboarding = !! window.calypsoifyGutenberg?.isGutenboarding;
 	const welcomeHeading = isGutenboarding
-		? __( 'Welcome to your new website' )
+		? __( 'Welcome to your site' )
```
Comment (razvanpapadopol) [inline apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js:55]: See https://github.com/Automattic/wp-calypso/pull/41043/files#r408130785

File: client/landing/gutenboarding/onboarding-block/design-selector/index.tsx [hunk 6]
```diff
@@ -50,11 +50,9 @@ const DesignSelector: React.FunctionComponent = () => {
 		<div className="design-selector">
 			<div className="design-selector__header">
 				<div className="design-selector__heading">
-					<Title>{ __( 'Choose a starting design' ) }</Title>
+					<Title>{ __( 'Choose a design' ) }</Title>
 					<SubTitle>
-						{ __(
-							'Get started with one of our top website layouts. You can always change it later'
-						) }
+						{ __( 'Pick your favorite homepage layout. You can customize or change it later.' ) }
```
Comment (razvanpapadopol) [inline client/landing/gutenboarding/onboarding-block/design-selector/index.tsx:55]: Screenshot 2020-04-14 at 15 39 36

File: client/landing/gutenboarding/onboarding-block/style-preview/index.tsx [hunk 7]
```diff
@@ -59,8 +59,10 @@ const StylePreview: React.FunctionComponent = () => {
 		<div className="style-preview">
 			<div className="style-preview__header">
 				<div className="style-preview__titles">
-					<Title>{ __( 'Select your fonts' ) }</Title>
-					<SubTitle>{ __( 'Add some personality to your design.' ) }</SubTitle>
+					<Title>{ __( 'Pick a font pairing' ) }</Title>
+					<SubTitle>
+						{ __( 'Add some personality to your design. You can fine-tune later.' ) }
+					</SubTitle>
```
Comment (razvanpapadopol) [inline client/landing/gutenboarding/onboarding-block/style-preview/index.tsx:65]: Screenshot 2020-04-14 at 15 39 52

**Comment by razvanpapadopol** [comment]
> Hi @razvanpapadopol -- I wanted to respond to the `Needs copy review` request, but I wasn't sure where to find the copy in question (or how to access /new in a way that would show it). Could you share how best to view it, and/or share screenshots here? Thanks in advance!

Hi @benhuberman!

Thanks for checking this out!

I've added screenshots for all the copy changes and you can access some of them using the [live link](https://hash-7dadf3c440095bc7a8dd6d0275a5cf0562000d50.calypso.live/new). 
- fill in the inputs on the first screen.
- press the button that appears below.
- you'll land on the [design selector ](https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408132090) screen.
- select one of the designs.
- you'll see the [style preview](https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408132725) step.

The welcome modal changes can't be previewed so easily but you can check the live modal by opening the editor for an existing site (eg: `https://wordpress.com/block-editor/page/[SITE_ADDRESS]/home`) and accessing _Welcome Guide_ in the sidebar
Screenshot 2020-04-14 at 15 57 12
Here we're going to use custom copy for users that have just created a site using `/new` flow.
step1: https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408130785
step2: https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408124507
step3: https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408125014

**Comment by benhuberman** [comment]
Thanks for these @razvanpapadopol, and sorry for the delay. I'll look at the copy by the end of the night (Pacific time).

**Comment by benhuberman** [comment]
Thanks again for the screenshots and instructions, @razvanpapadopol. Here are some notes and suggested edits:

In the "Customize your content" [modal](https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408124507), I'd swap "pre-made" with "existing," i.e. "Start with an existing layout and make it your own."

In the "Add anything you want" [modal](https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408125014), I suggest:
- Tweak the heading to "Add (almost) anything" -- I think we could use a slight hedge here, since some users are bound to find things they want to add that they can't.
- In the description, I'd tweak the final sentence to this: "Rearrange the blocks on your page until they're just right."

In the "Welcome to your site" [modal](https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408130785), I suggest we turn "site" in the heading to "website." (I saw this was recently changed, and that it had originally been "website." If there's a specific reason for the switch I'm not aware of, no worries, but I think it's better to use "website" in the heading, and "site" in the description copy.) 

I also suggest we slightly tweak the description copy: "Edit your homepage, add the pages you need, and change your site's look and feel." 

The copy on the "Choose a design" screen looks fine to me.

In the "Pick a font pairing" [screen](https://github.com/Automattic/wp-calypso/pull/41043#discussion_r408132725), I suggest a redo; I think "add personality" feels off here in connection to fonts. How about... "Customize your design with typography. You can always fine-tune it later."

Does this cover all the included screens/copy? Thanks for letting me know if I've missed anything along the way and I'll jump back into this as quickly as possible.

**Comment by sirreal** [comment]
Rebased and applied all the suggested changes from https://github.com/Automattic/wp-calypso/pull/41043#issuecomment-614443193 in 46dbf4115d4d9813328aa9127370f4124d244b0f

**Comment by akirk** [comment]
Could you please update your description with a screenshot that shows the changed / introduced strings? Thanks!

**Comment by razvanpapadopol** [comment]
Hi @akirk ! 👋 Sorry for missing this part. Updated the description now with latest screenshots of the changes grouped by file so you can ignore all the comments. I hope everything is clear now.

**Comment by akirk** [comment]
Great, thank you!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3337654

Thank you @razvanpapadopol for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3337654).

**Pull request merged**